pkgs/os-specific/darwin/swiftbar/default.nix

```diff
@@ -41,5 +43,6 @@ stdenvNoCC.mkDerivation rec {
     platforms = platforms.darwin;
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     maintainers = with maintainers; [ matteopacini ];
+    hydraPlatforms = [ ];
```

Comment:
Whats your motivation to set this?

Reply:
Was told in a a previous PR that it doesn't make sense to have have Hydra caching binary distributions.

Happy to revert this if doesn't make sense 👍🏻